sdk/containerinstance/Microsoft.Azure.Management.ContainerInstance/src/Generated/ContainersOperations.cs

```diff
@@ -76,6 +76,42 @@ internal ContainersOperations(ContainerInstanceManagementClient client)
         /// <param name='cancellationToken'>
         /// The cancellation token.
         /// </param>
+        /// </return>
+        public async Task<AzureOperationResponse<Logs>> ListLogsWithHttpMessagesAsync(string resourceGroupName, string containerGroupName, string containerName, int? tail = default(int?), Dictionary<string, List<string>> customHeaders = null, CancellationToken cancellationToken = default(CancellationToken))
+        {
+            return ListLogsWithHttpMessagesAsync(resourceGroupName, containerGroupName, containerName, tail: tail, timestamps: false, customHeaders: customHeaders, cancellationToken: cancellationToken);
```

Comment:
It looks like you added this method directly onto the generated Operation class.  What you need to do to make this customization is the following:

(1) This method signature appears in IContainersOperations, ContainersOperations, and ContainersOperationsExtensions, and all 3 of these would need the new overload(s)

(2) We don't edit the generated code, as these methods would go away after the next round of generation.  Instead,  since the generated classes and interfaces are partial, we add a new class with just the new method signatures in a partial class with the same name in src/Customizations.  This will allow this customnization to continue through successive updates.

(3) The build is failing because for the async method to return Task<AzureOperationResponse<Logs>> you need to return an AzureOperationResponse<Logs>, and it will automatically be wrapped.  To do this, you would either mark this method as not async, or you would await the result of the call to the other method:
``` C#
return await ListLogsWithHttpMessagesAsync(resourceGroupName, containerGroupName, containerName, tail: tail, timestamps: false, customHeaders: customHeaders, cancellationToken: cancellationToken);
```

Reply:
Since you aren't actually awaiting any calls, it is fine to remove the async keyword, without making the change above.

Reply:
@markcowl I moved the overload into customizations as a partial class. The examples really helped!